### ISSUE TITLE
lets check your model for over fitting per this paper many models have over fitting 

### ISSUE THREAD

**Issue by Sandy4321** [yoonkim/CNN_sentence#15]
new drop out for embeddings layer : conventional drop outs leads to overfititng
http://arxiv.org/abs/1512.05287
A Theoretically Grounded Application of Dropout in Recurrent Neural Networks

Yarin Ga
